Program.cs

```diff
@@ -21,12 +32,20 @@ public static void Main(string[] args)
 
             XmlDocument projectFile = LoadProjectFile(projectFilePath);
 
-            string[] files = Directory.EnumerateFiles(projectPath, "*", SearchOption.AllDirectories)
+            IEnumerable<string> filesQuery = Directory.EnumerateFiles(projectPath, "*", SearchOption.AllDirectories)
                 .Where(path => path != projectFilePath)
-                .Select(path => path.Split(new[] {projectPath}, 2, StringSplitOptions.RemoveEmptyEntries))
-                .Select(strings => strings[0])
-                .Select(path => path.TrimStart('\\'))
-                .ToArray();
+                .Select(path => RemovePrefix(path, projectPath))
```

Comment:
This came up in CI testing -- `Split`'s `count` parameter limits the number of *returned* substrings after empty strings have been removed. `"CovertInfiltration/Src/CovertInfiltration/Classes/CI_DataStructures.uc"` with a project directory of `"CovertInfiltration/"` will be split into `["Src/", "Classes/abc.uc"]` which makes the previous logic consider `"Src/"` a (missing) file name.